### PR TITLE
fix some zh translate

### PR DESCRIPTION
**Pull request by helight**
fix some zh translate

### CODE REVIEW

File: i18n/zh/messages.zh.xlf [hunk 1]
```diff
@@ -3264,7 +3264,7 @@
       </trans-unit>
       <trans-unit id="2244c094c4409a5d5bb05ae8a079db563b730d09" datatype="html">
         <source>A secret with the specified name will be added to the cluster in the namespace.</source>
-        <target>具有指定名称的 secret 将添加到命名空间中的集群中。</target>
+        <target>将添加指定名称的 secret 到命名空间中的集群中。</target>
```
Comment (zehuaiWANG) [inline i18n/zh/messages.zh.xlf]: 最好保持 Name 的翻译一致，要么翻译成 “名字” 要么翻译成 “名称”.

**Comment by zehuaiWANG** [comment]
thanks for @helight 's contribution, most of them look good. And I think it‘s better to keep the translation of “Name” consistent.

File: i18n/zh/messages.zh.xlf (hunk 1 above)
Comment (helight) [reply i18n/zh/messages.zh.xlf]: 有些地方 “名字” 比较通顺一点，有些地方“名称”比较通顺一点。所以没有替换。

Comment (zehuaiWANG) [reply i18n/zh/messages.zh.xlf]: @hwdef hi~ 这里确实有这个问题，有些地方用"名字"比较好，有些地方用“名称”比较好。你怎么看？

Comment (hwdef) [reply i18n/zh/messages.zh.xlf]: 我同意 @zehuaiWANG 的看法，而且我觉得“名称”比较正式，用名称比较好。
/assign @tanjunchen 你怎么看？

File: i18n/zh/messages.zh.xlf [hunk 2]
```diff
@@ -108,7 +108,7 @@
       </trans-unit>
       <trans-unit id="d984832653b92d083bf8ca529c4d99ab55a047ef" datatype="html">
         <source>Size: <x id="INTERPOLATION" equiv-text="{{loaded | kdMemory}}"/> B</source>
-        <target>尺寸： <x id="INTERPOLATION" equiv-text="{{loaded | kdMemory}}"/> B</target>
+        <target>内存大小： <x id="INTERPOLATION" equiv-text="{{loaded | kdMemory}}"/> B</target>
```
Comment (tanjunchen) [inline i18n/zh/messages.zh.xlf]: great job

File: i18n/zh/messages.zh.xlf (hunk 1 above)
Comment (tanjunchen) [reply i18n/zh/messages.zh.xlf]: 我比较认同名称

Comment (helight) [reply i18n/zh/messages.zh.xlf]: 好的，那就统一都修改为名称

**Comment by hwdef** [comment]
/approve
Thanks for your contribution.

**Comment by tanjunchen** [comment]
/LGTM
Thanks for your contribution.

**Comment by zehuaiWANG** [comment]
```
[0] level=warning msg="[runner] Can't run linter goanalysis_metalinter: ineffassign: analysis skipped: errors in package: [/home/travis/build/kubernetes/dashboard/src/app/backend/plugin/client/clientset/versioned/clientset.go:1: no such file or directory /home/travis/build/kubernetes/dashboard/src/app/backend/plugin/client/clientset/versioned/doc.go:1: no such file or directory]"
[0] level=error msg="Running error: ineffassign: analysis skipped: errors in package: [/home/travis/build/kubernetes/dashboard/src/app/backend/plugin/client/clientset/versioned/clientset.go:1: no such file or directory /home/travis/build/kubernetes/dashboard/src/app/backend/plugin/client/clientset/versioned/doc.go:1: no such file or directory]"
```

@floreks @shu-mutou hi~ I found out that CI has such problem, it seems that it is not caused by this PR, but I am not familiar with this one, could you help to find out why this happened?

**Comment by shu-mutou** [comment]
@zehuaiWANG I've done `Restart job` on travis UI and passed the test.

**Pull request merged**